keyboards/jj50/keymaps/archetype/keymap.c

```diff
@@ -0,0 +1,238 @@
+/*
+Base Copyright 2017 Luiz Ribeiro <luizribeiro@gmail.com>
+Modified 2017 Andrew Novak <ndrw.nvk@gmail.com>
+Modified 2018 Wayne Jones (WarmCatUK) <waynekjones@gmail.com>
+Modified 2019 Iason Dimitrakopoulos (idimitrakopoulos) <idimitrakopoulos@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public LicensezZZ
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+#include "action_layer.h"
+#include "rgblight.h"
+#include "quantum.h"
+
+#define ______ KC_TRNS
```

Comment:
By including `#include QMK_KEYBOARD_H`, you get access to two `#defines`. They are a 7 space `KC_TRNS` and a `7X KC_NO`. Unless you have specific use cases for the 6 space `KC_TRNS` like you defined here, I'd prefer you use the predefined ones. 

```
_______ = KC_TRNS
XXXXXXX = KC_NO
```

Reply:
You're right didn't know that was included there. I have removed my defines to make the code simpler